evm/hyperliquid.js

```diff
@@ -80,30 +80,22 @@ async function updateBlockSize(wallet, chain, options) {
     return result;
 }
 
-async function deployer(wallet, chain, options) {
-    const [tokenId] = options.args;
+async function deployer(wallet, chain, args) {
```

Comment:
```suggestion
async function deployer(wallet, chain, args, _options) {
```
since the caller is still passing in `options` even if we don't use it. same everywhere